build.gradle

```diff
@@ -17,26 +17,26 @@
 buildscript {
     ext.kotlin_version = '1.4.10'
     repositories {
-        jcenter()
         google()
+        jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.0-alpha16'
-        classpath 'com.google.gms:google-services:4.3.3'
+        //classpath 'com.google.gms:google-services:x.y.z'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.27.2'
     }
 }
 
 allprojects {
     repositories {
+        google()
         jcenter()
         maven { url "https://jitpack.io" }
-        maven { url 'https://maven.fabric.io/public' }
-        google()
+        maven { url "https://maven.fabric.io/public" }
     }
 }
 
-//task clean(type: Delete) {
```

Comment:
I actually have these commented out as it's clashing with the clean task from spotless.
https://stackoverflow.com/a/59026437

Reply:
🤔 I see... I'll undo this, then.